core/src/utils/config.ts

```diff
@@ -18,10 +18,18 @@ export interface IonicConfig {
   rippleEffect?: boolean;
 
   /**
-   * The mode determines which platform styles to use for the whole application.
+   * The mode determines the platform behaviors to use for the whole application.
+   *
+   * Without a theme specified, mode will fallback to determining which platform
+   * styles to use for the whole application.
```

Comment:
```suggestion
   * The mode determines the platform behaviors to use for the entire application.
   *
   * Without a specified theme, the mode will be used to determine which platform
   * styles to use throughout the application.
```

I don't know if this is what you are trying to say, I am a bit confused here.